crates/nu-protocol/src/errors/shell_error.rs

```diff
@@ -1097,6 +1097,11 @@ pub enum ShellError {
         span: Span,
     },
 
+    /// This is a generic error type used for user and plugin-generated errors.
+    #[error(transparent)]
+    #[diagnostic(transparent)]
+    LabeledError(#[from] super::LabeledError),
```

Comment:
Might be worth boxing: currently `ShellError` takes 120 bytes, `LabeledError` takes 144 bytes alone, so this blows it up to 152 bytes (and to the next cache line)